server/models/schema.sql

```diff
@@ -2184,6 +2192,7 @@ CREATE TABLE `service_fee_center` (
 CREATE TABLE `tags`(
   `uuid` BINARY(16) NOT NULL PRIMARY KEY,
   `name` VARCHAR(100) NOT NULL,
+  `color` VARCHAR(20) NULL,
```

Comment:
Since you are adding this property, I would go ahead and make it much longer.  VARCHAR(20) is super limiting.